src/modules/verifiableCredentials/verifiableCredentials.types.ts

```diff
@@ -0,0 +1,54 @@
+import { TypedDataField } from '@ethersproject/abstract-signer';
+
+export interface VerifiableCredential<T> {
+  '@context': string | string[];
+  id?: string;
+  issuer: string;
+  type: string[];
+  credentialSubject: T;
+}
+
+export interface Domain {
```

Comment:
it is already defined in `ethers` https://github.com/ethers-io/ethers.js/blob/dfaa8ee7e6f808dc7f363863243ea650bc629197/packages/abstract-signer/src.ts/index.ts#L25